internal/pkg/cli/pipeline_init_test.go

```diff
@@ -607,52 +607,18 @@ func TestInitPipelineOpts_Execute(t *testing.T) {
 	}
 }
 
-func TestInitPipelineOpts_createSecretName(t *testing.T) {
```

Comment:
Any reason we don't want to keep these?

Reply:
`TestInitPipelineOpts_Execute` is already testing if the secret is created with the appropriate name :)